VTEX - Catalog API.json

```diff
@@ -18414,7 +17643,7 @@
                     "CommercialConditionId": {
                         "type": "integer",
                         "format": "int32",
-                        "description": "SKU Commercial Condition ID."
+                        "description": "SKU commercial condition ID."
```

Comment:
```suggestion
                        "description": "Commercial condition ID, used to define SKU specific promotions or installment rules. In case of no specific condition, use `1` (default value). This field does not accept `0`. Learn more at [Registering a commercial condition](https://help.vtex.com/tutorial/registering-a-commercial-condition--tutorials_445).",
                        "default": 1
```